python/pyspark/sql/tests/pandas/test_pandas_transform_with_state.py

```diff
@@ -399,21 +431,30 @@ def handleInputRows(self, key, rows) -> Iterator[pd.DataFrame]:
             iter = self.ttl_list_state.get()
             for s in iter:
                 ttl_list_state_count += s[0]
+        if self.ttl_map_state.exists():
+            ttl_map_state_count = self.ttl_map_state.get_value(key)[0]
```

Comment:
just to double confirm, here the key does not necessarily need to be the same with grouping key, right? It's just to simplify the test.

Reply:
Yep, correct.